### ISSUE TITLE
AccessToken_utils.ps1 - Cannot bind argument to parameter 'Body' because it is an empty string.

### ISSUE THREAD

**Issue by spaffhazz** [Gerenios/AADInternals#94]
```
┌──(user㉿kali)-[/home/user]
└─PS> Get-AADIntAccessTokenForAzureCoreManagement -SaveToCache
    ___    ___    ____  ____      __                        __    
   /   |  /   |  / __ \/  _/___  / /____  _________  ____ _/ /____
  / /| | / /| | / / / // // __ \/ __/ _ \/ ___/ __ \/ __ `/ / ___/
 / ___ |/ ___ |/ /_/ _/ // / / / /_/  __/ /  / / / / /_/ / (__  ) 
/_/  |_/_/  |_/_____/___/_/ /_/\__/\___/_/  /_/ /_/\__,_/_/____/  
  
 v0.9.3 by @DrAzureAD (Nestori Syynimaa)
Logging in to Microsoft Services                                                                                        
Enter email, phone, or Skype: redacted
Password: ***************
Parse-LoginMicrosoftOnlineComConfig: /home/user/.local/share/powershell/Modules/AADInternals/0.9.3/AccessToken_utils.ps1:2741
Line |
2741 |  … $config = Parse-LoginMicrosoftOnlineComConfig -Body $response.Content
     |                                                        ~~~~~~~~~~~~~~~~~
     | Cannot bind argument to parameter 'Body' because it is an empty string.

```

I'm getting this error whenever I try to get access tokens. Not sure how to fix this. Tried updating AADInternals but it didnt help and I'm on the latest one right now.

**Comment by jeh0753** [comment]
Got this error as well on `Get-AADIntAccessTokenForAzureCoreManagement -SaveToCache` command

**Comment by spaffhazz** [comment]
@jeh0753 were you able to resolve the issue on your end?

**Comment by weyCC81** [comment]
I have the same issue with Version 0.9.3 and 0.9.4, this includes the following commands:

```
Get-AADIntAccessTokenForAADGraph -Credentials $cred
Get-AADIntAccessTokenForAADGraph -Credentials $cred -SaveToCache
Get-AADIntAccessTokenForAADGraph -Credentials $cred_a -SaveToCache -OTPSecretKey $totp_a
Get-AADIntAccessTokenForAADGraph -Credentials $cred_a -SaveToCache -OTPSecretKey $totp_a2
Get-AADIntAccessTokenForMSGraph -Credentials $cred
Get-AADIntAccessTokenForMSGraph -Credentials $cred -SaveToCache
Get-AADIntAccessTokenForMSGraph -Credentials $cred_a -SaveToCache -OTPSecretKey $totp_a
Get-AADIntAccessTokenForMSGraph -Credentials $cred -SaveToMgCache # just with 0.9.4 possible
```

**Comment by weyCC81** [comment]
May related issue: Fix WebCmdlets when -Body is specified but ContentType is not
> https://github.com/PowerShell/PowerShell/pull/24145
> https://github.com/PowerShell/PowerShell/pull/23952

Tried fix: Upgrade to PowerShell 7.4.5 
> https://github.com/PowerShell/PowerShell/releases/tag/v7.4.5

**Result: Upgrade did not fix the issue**
